### DISCUSSION TITLE
Where should ML be used rather than (or in combination with) physics-based models?

### DISCUSSION THREAD

**Discussion by samyoung-dsci** [alan-turing-institute/ADViCE#90] [Key sector: Built Environment]
There was lots of discussion on the webinar around physics-based models and ML, including the question: "We've had tremendous success using physics-based models to do much if not most of the benefits of heat pump optimisation, flow temperature control, and optimising for weather forecast and electricity price trade-offs, consistently delivering substantial running cost savings. Is it important whether or not a model is physics-based or 'black box AI'?"

I think a lot of people in the buildings space lean quite strongly towards physics-based models, so I'm interested where people think ML/AI should replace (or augment) physics-based models.

**Comment by samyoung-dsci** [comment]
In my mind, the assumptions in a physics-based models are a double-edge sword. They avoid completely unreasonable physical scenarios that an AI might not realise are unrealistic, and also make it much easier to model with little data/compute. However, they also mean that when those assumptions are incomplete real world outcomes can deviate quite significantly from the model. It can be impractical to make the physics-based models more and more sophisticated to capture all the edge cases (and more and more data can be required), so at some point a more ML/AI approach can be more effective. 

This is well reflected in [Google's ML guide rules 1-3](https://developers.google.com/machine-learning/guides/rules-of-ml#before_machine_learning)

**Comment by willrrowe** [comment]
When sizing heat pumps, it seems like the "educated guess" for the building fabric is the biggest uncertainty. It seems smart meter data could be used to get a more accurate estimate of whole home energy use, and then likely building fabric could be fit to the home using this data. Then the model of the home would be accurate, and radiators could then be sized based on actual usage data (rather than best guess)
